games/sudoku-solver/Sudoku.java

```diff
@@ -0,0 +1,53 @@
+import java.util.*;
+class Sudoku {
+  public void sudokuSolver(int[][] board) {
```

Comment:
nit : poor naming. Please rename this to generateBoard()

Reply:
Agree, fixed here e18c92a